src/components/Section.js

```diff
@@ -4,7 +4,7 @@ export default class Section {
     this._renderer = renderer;
     this._element = document.querySelector(selector);
   }
-  renderItems() {
+  renderItems(item) {
     this._items.forEach((item) => {
       this._renderer(item);
```

Comment:
* You haven't corrected my remark https://github.com/omegax117/se_project_aroundtheus/pull/1#discussion_r1671744255  You need to use `renderItems` for rendering the array of cards. 


```js
  renderItems(items) {
    items.forEach((item) => {
      this._renderer(item);
    });
```